File: Java_Programs/Java_Lab_Program/Lab_5/Program_4.java
// Write a Java program to copy all the elements of an array to another array and print
// both the array elements.

public class Program_4 {
     public static void main(String[] args) {
        
          int array[]={10,20,30,40,50};

          Program_4 p4=new Program_4();

           int revese_arr[]=new int[array.length];  

          revese_arr=p4.copy_array(array);

          for(int i:revese_arr){
            System.out.print(i+" ");
          }

     }
     
        public static int[] copy_array(int array[]){

              int array2[]=new int[array.length];

                for(int i=0; i<array.length; i++){
                     array2[i]=array[i];
                }
                return array2;
        } 

}

File: C_DSA_Lab/LAB_2/Program_4.c
// Write a program to copy all the elements of one array to another array. 

#include<stdio.h>

int main(){

    int array_size;

    printf("Enter Array Size : ");
    scanf("%d",&array_size);

    int array1[array_size];
    int copy_arr[array_size];

    for(int i=0; i<array_size; i++){
         printf("Enter Array Element : %d ",(i+1));
         scanf("%d",&array1[i]);
         copy_arr[i]=array1[i];
    }

    printf("\n------------------\n");
    for(int i=0; i<array_size; i++){
          printf(" %d ",copy_arr[i]);
    }
    printf("\n------------------\n");
}